addons/web_editor/static/lib/odoo-editor/test/spec/odoo.test.js

```diff
@@ -0,0 +1,21 @@
+import { BasicEditor, testEditor, unformat } from '../utils.js';
+
+describe('Odoo', () => {
```

Comment:
I suggest calling this section either "Fields" or "Odoo Fields", but not just "Odoo". Once you have chosen one of those two options, you can update the name of this file too then I can r+ the PR.

Reply:
The reason I've called this file Odoo is because I don't think we will write so many specific code for Odoo in the odoo-editor folder so we could write any specific Odoo tests in this file. If this file is called Odoo, I think the first describe should be called the same way.

With that in mind, do you still want to rename this file and describe to be more specific ?

Reply:
Considering the editor is not an actual lib anymore, it seems weird to me to have a test file called `odoo.test.js`. It's basically the same as having, like `addons/accounting/tests/odoo.test.js`. Don't you think it is weird ? What accounting feature are they testing in there ? Odoo accounting features ? But aren't all the features _Odoo_ features ? Or the interaction of the accounting module with Odoo ? But it's an Odoo module so, it's not like it's supposed to interface with _other_ ERP systems... You see what I mean ? Personally, I'd call the file `fields.test.js` but I'm open to other suggestions.

Reply:
We did not started from the same premise. I still consider the Odoo editor as a standalone editor that could be instanced outside Odoo. I think that distinction is relevant if we want to make the editor modular.